### PR TITLE
Add human readable test feedback

### PR DESCRIPTION
**Pull request by fapdash**
After I failed to read the test output I decided to make it clearer.

### CODE REVIEW

File: bin/run-tests.sh [hunk 1]
```diff
@@ -34,8 +36,19 @@ for test_dir in tests/*; do
     diff "${results_file_path}" "${expected_results_file_path}"
 
     if [ $? -ne 0 ]; then
+        echo "${test_dir_name}: ERROR: comparison failed - results.json not matching expected_results.js"
```
Comment (ErikSchierboom) [inline bin/run-tests.sh:39]:
```suggestion
        echo "${test_dir_name}: ERROR: comparison failed - results.json not matching expected_results.json"
```

File: bin/run-tests.sh [hunk 2]
```diff
@@ -34,8 +36,19 @@ for test_dir in tests/*; do
...
         exit_code=1
+        test_error_counter=$[$test_error_counter + 1]
+    else
+        echo "${test_dir_name}: SUCCESS: results.json is matching expected_results.json"
     fi
+    test_counter=$[$test_counter + 1]
 done
 
+if [ "${exit_code}" -eq 0 ]; then
+    echo "SUCEESS: All tests passed"
```
Comment (ErikSchierboom) [inline bin/run-tests.sh:49]:
```suggestion
    echo "SUCCESS: All tests passed"
```

File: bin/run-tests.sh [hunk 3]
```diff
@@ -1,7 +1,7 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
```
[the remark this answers is not in the document]
Comment (ErikSchierboom) [reply bin/run-tests.sh:1]: That perfectly fine. Feel free use `bash` in the other scripts, but preferrably in a different PR.

File: bin/run-tests.sh [hunk 4]
```diff
@@ -12,6 +12,8 @@
 # ./bin/run-tests.sh
 
 exit_code=0
+test_counter=0
```
Comment (ErikSchierboom) [inline bin/run-tests.sh:15]: Maybe change this to:

```suggestion
test_success_counter=0
```

Note: you'd also need to update this where it is used

File: bin/run-tests.sh (hunk 3 above)
[the remark this answers is not in the document]
Comment (fapdash) [reply bin/run-tests.sh:1]: Would you prefer something like the white space commit in an extra PR as well?

**Review by ErikSchierboom — APPROVED** [review]

**Comment by ErikSchierboom** [comment]
Thanks!

**Pull request merged**